app/services/video-encoding-optimizations/index.ts

```diff
@@ -0,0 +1,110 @@
+import { Service } from 'services/service';
+import { SettingsService, ISettingsSubCategory } from 'services/settings';
+import { StreamingService } from 'services/streaming';
+import { Inject } from '../../util/injector';
+import { IProfile, IEncoderPreset, getPreset } from './definitions';
+
+const cloneDeep = require('lodash.clonedeep');
+
+let previousSettings: any;
+let isUsingEncodingOptimizations: boolean = false;
```

Comment:
Not a big deal, but the `boolean` annotation is redundant since you are immediately assigning a boolean value to it.

Reply:
Also, I would define `previousSettings` and `isUsingEncodingOptimizations` as attributes on the class instead of global variables in the module.

Reply:
Done.